testsuite/src/it/java/org/graalvm/tests/integration/utils/GDBSession.java

```diff
@@ -233,12 +235,13 @@ public CP[] get(boolean inContainer) {
     DEBUG_QUARKUS_FULL_MICROPROFILE {
         @Override
         public CP[] get(boolean inContainer) {
+            // The huge timeout is needed because it takes a very long time to set a breakpoint, even after: https://github.com/graalvm/mandrel/pull/545
+            final int increasedTimeout = (UsedVersion.getVersion(inContainer).compareTo(Version.create(23, 0, 0)) >= 0) ? 60 : DEFAULT_TIMEOUT_SECONDS;
```

Comment:
Could you please do me a favour and measure the time it takes for the breakpoint to hit with something like this:

```
gdb -ex 'set breakpoint pending on' -ex 'set debuginfod enabled off' -ex "set \$starttime=$(date +%s.%N)" -ex 'b ConfigTestController.java:33' -ex 'shell echo set \$endtime=$(date +%s.%N) > ~/gdbtmp.txt' -ex 'source ~/gdbtmp.txt' -ex 'print $endtime-$starttime'  quarkus-runner
```

I'd like to see which numbers we get on various systems.